README.md

```diff
@@ -1,29 +1,13 @@
-<!-- Update this title with a descriptive name. Use sentence case. -->
-# Terraform modules template project
+# IBM Cloud Activity Tracker
```

Comment:
This module is about Activity Tracker Event Routing not about Activity Tracker.
https://cloud.ibm.com/docs/atracker?topic=atracker-getting-started&interface=ui

Reply:
Check all the related docs and variable description accordingly, and also be consistent everywhere for the name.
Activity Tracker 
Activity Tracker Event Routing
Activity Tracker event routing
Refer doc (link provided in the previous comment) and make changes accordingly.